diabetes_regression/register/register_model.py

```diff
@@ -109,35 +136,28 @@ def model_already_registered(model_name, exp, run_id):
 
 
 def register_aml_model(
+    model_path,
     model_name,
     exp,
     run_id,
     build_id: str = 'none',
     build_uri=None
 ):
     try:
-        if (build_id != 'none'):
-            model_already_registered(model_name, exp, run_id)
-            run = Run(experiment=exp, run_id=run_id)
-            tagsValue = {"area": "diabetes_regression",
-                         "BuildId": build_id, "run_id": run_id,
-                         "experiment_name": exp.name}
-            if (build_uri is not None):
-                tagsValue["BuildUri"] = build_uri
-        else:
-            run = Run(experiment=exp, run_id=run_id)
-            if (run is not None):
-                tagsValue = {"area": "diabetes_regression",
-                             "run_id": run_id, "experiment_name": exp.name}
-            else:
-                print("A model run for experiment", exp.name,
-                      "matching properties run_id =", run_id,
-                      "was not found. Skipping model registration.")
-                sys.exit(0)
+        model_already_registered(model_name, exp, run_id)
+        tagsValue = {
+            "area": "diabetes_regression",
+            "BuildId": build_id,
```

Comment:
This value may not exist.  The logic you removed prevented logging a 'none' build_id from line 101.

Reply:
See updated logic